### PR TITLE
Add Openstack provider

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,8 +7,9 @@ require (
 	github.com/pulumi/pulumi-azure-native-sdk/resources/v2 v2.10.0
 	github.com/pulumi/pulumi-azure-native-sdk/storage/v2 v2.10.0
 	github.com/pulumi/pulumi-gcp/sdk/v6 v6.65.0
+	github.com/pulumi/pulumi-openstack/sdk/v3 v3.15.1
 	github.com/pulumi/pulumi-tls/sdk/v4 v4.11.0
-	github.com/pulumi/pulumi/sdk/v3 v3.87.0
+	github.com/pulumi/pulumi/sdk/v3 v3.103.1
```
Comment (adrianriobo) [inline go.mod:12]: there is a miss match between this sdk version and the cli version on the Containerfile

Comment (praveenkumar) [reply go.mod:12]: Will update that

Comment (praveenkumar) [reply go.mod]: updated same as in container file v3.107.0

File: README.md [hunk 2]
```diff
@@ -63,6 +71,18 @@ gcloud compute images create crc --source-uri  gs://crc-bundle-github-ci/crc.tar
 gcloud compute images list --no-standard-images | grep crc
 ```
 
+As of now please use following manual steps to import the image on `openstack`:
+```bash
+# Download the required bundle from https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/crc/bundles/openshift/
+curl -L -O https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.14.3/crc_libvirt_4.14.3_amd64.crcbundle
+# Extract the bundle
+tar -xvf crc_libvirt_4.14.3_amd64.crcbundle && cd crc_libvirt_4.14.3_amd64
```
Comment (adrianriobo) [inline README.md:79]: Is this working for you? I need to `unzstd` before doing `tar -xvf...`

Comment (adrianriobo) [reply README.md:79]: I see, so it seems if zstd is installed tar may work out of the box, if not it will fail

Comment (praveenkumar) [reply README.md:79]: Right

**Comment by adrianriobo** [comment]
Tested with version `4.14.12`and it is working perfectly

**Pull request merged**